### ISSUE TITLE
Multisite "White Label" Site Creation Wizard

### ISSUE THREAD

**Issue by sethrubenstein** [PixelRocket-Biz/cliff-michaels#210]
Cliff would like end users to be able to go through a wizard, pick colors, text, courses they want added etc... and checkout and a new site be spun up for them.

**Comment by sethrubenstein** [comment]
@cliffmichaelsgithub I've started work on this - would you also like me to take care of these 2 items while working on this functionality for the White Label system?

1. https://github.com/PixelRocket-Biz/cliff-michaels/issues/131
2. https://github.com/PixelRocket-Biz/cliff-michaels/issues/111

**Comment by cliffmichaelsgithub** [comment]
Confused what these are. I don't always understand how to interface github on cell phone. Hard to follow your notes sometimes. Can we talk 5 minutes today?


On Feb 16, 2015, at 7:38 AM, Seth Rubenstein <notifications@github.com<mailto:notifications@github.com>> wrote:


@cliffmichaelsgithub<https://github.com/cliffmichaelsgithub> I've started work on this - would you also like me to take care of these 2 items while working on this functionality for the White Label system?

  1.  #131<https://github.com/PixelRocket-Biz/cliff-michaels/issues/131>
  2.  #111<https://github.com/PixelRocket-Biz/cliff-michaels/issues/111>

-

**Comment by sethrubenstein** [comment]
I'm available between 3:30 and 4:15 today hard stop at 4:15.

**Comment by cliffmichaelsgithub** [comment]
Thanks. Just need 5-10 mins to clarify and make sure we're on same page. Also have to watch my budget carefully

Cliff > 310-488-6422
VC | Mortgages | Private Equity
Real Estate Investments
CliffMichaels.com<http://cliffmichaels.com/funding><http://cliffmichaels.com/funding>/<http://cliffmichaels.com/funding>Funding<http://cliffmichaels.com/funding>
Coach | Speaker | Facilitator
E-Courses | Forums | Retreats
Career & Entrepreneurial Training
CliffMichaels.com<http://cliffmichaels.com/> | LinkedIN<http://linkedin.com/in/cliffmichaels>









On Feb 16, 2015, at 8:29 AM, Seth Rubenstein <notifications@github.com<mailto:notifications@github.com>> wrote:


I'm available between 3:30 and 4:15 today hard stop at 4:15.

-

**Comment by cliffmichaelsgithub** [comment]
Subheader of some kind would be titled: Custom Edition Features & Benefits 

Your Custom Header (School, Club, Company) 
Choose any of Cliff's E-Courses + Add Your Own
Choose any of Cliff's Tools + Add or Customize Your Own
Progress & Management Dashboard to Monitor Results
Add a Chat Room or Instructor's Message Board
Customize Quizzes + Pass / Fail Criteria

**Comment by sethrubenstein** [comment]
Note to self: Create product type in WooCommerce for this. Also should create new product type for courses merge back into Apollo.

**Comment by sethrubenstein** [comment]
- [ ] Note to self: Need to use GF Api to create a form for setting up site on first launch. Check for if is admin and a get_option key. Choose colors... etc.... When completed do gf hook and add_option to options table that says site_setup_complete: true.
- [ ] Create a sep form that will let the user add products or add all products into their library.

**Comment by cliffmichaelsgithub** [comment]
What time Thursday night? How's wizard coming?

**Comment by sethrubenstein** [comment]
See my reply to your email regarding Thursday. Wizard will be showable to you Friday evening. I have everything done except for the actual form the white label purchaser will fill out on their site to setup all the details - working on that now.

**Comment by cliffmichaelsgithub** [comment]
Might have missed it or deleted. May be out Friday. Weekend?

**Comment by sethrubenstein** [comment]
Yea, I can do Saturday evening or Sunday afternoon/evening.

**Comment by cliffmichaelsgithub** [comment]
Let's do Sunday

Cliff | 310-488-6422
VC | Start-Up | Entertainment
Private & Real Estate Finance
Personal & Business Consulting
CliffMichaels.com<http://cliffmichaels.com/funding><http://cliffmichaels.com/funding>/<http://cliffmichaels.com/funding>Funding<http://cliffmichaels.com/funding>
CliffMichaels.com<http://cliffmichaels.com/> | LinkedIN<http://linkedin.com/in/cliffmichaels>









On Apr 15, 2015, at 7:51 PM, Seth Rubenstein <notifications@github.com<mailto:notifications@github.com>> wrote:


Yea, I can do Saturday evening or Sunday afternoon/evening.

-

**Comment by cliffmichaelsgithub** [comment]
I hope to look again end of month when this is ready. Is the wizard / tutorial ready or needs something from me?

Cliff | 310-488-6422
VC | Start-Up | Entertainment
Private & Real Estate Finance
Personal & Business Consulting
CliffMichaels.com<http://cliffmichaels.com/funding><http://cliffmichaels.com/funding>/<http://cliffmichaels.com/funding>Funding<http://cliffmichaels.com/funding>
CliffMichaels.com<http://cliffmichaels.com/> | LinkedIN<http://linkedin.com/in/cliffmichaels>









On Jun 17, 2015, at 10:13 AM, Seth Rubenstein <notifications@github.com<mailto:notifications@github.com>> wrote:


Closed #210<https://github.com/PixelRocket-Biz/cliff-michaels/issues/210>.

-
